setup.cfg

```diff
@@ -9,12 +9,17 @@ sections = FUTURE,STDLIB,FLASK,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
 not_skip = __init__.py
 
 [pylint]
-disable = missing-docstring
-extension-pkg-whitelist=lxml
+disable = missing-docstring,no-member,redefined-builtin,redefined-outer-name,unused-variable
```

Comment:
It'd be worth extracting the CS changes.

Reply:
This will result in having to handle them to then remove them. If the service was more mature I would agree but I think it's ok for now. Do you still want me to extract these?

Reply:
@thewilkybarkid ^^^

Reply:
Mean open a new PR with just those changes, merge that, then updating this branch reduces the size of the diff.

Reply:
> Mean open a new PR with just those changes, merge that, then updating this branch reduces the size of the diff.

https://github.com/libero/search/pull/10